defaults/main.yml

```diff
@@ -4,3 +4,6 @@ server:
 php:
     ppa: "php5-5.6"
     packages: ["php5-cli", "php5-intl", "php5-mcrypt"]
+
+php_version: "{% if php.ppa == 'php-7.0' %}php7.0{% else %}php5{% endif %}"
+php_config_prefix: "{% if php.ppa == 'php-7.0' %}/etc/php/7.0{% else %}/etc/php5{% endif %}"
```

Comment:
@debo, I think that moving php configs from default directories is dangerous and it can confusing people that want to check php-7.0 work and install it on their servers. For example, I installed php-7.0 and now I know that I should check all configs and scripts for php5 references.

Conditionals in defaults/main.yml used in most starred nginx role [jdauphant/ansible-role-nginx](https://github.com/jdauphant/ansible-role-nginx/blob/master/defaults/main.yml).

I have no idea how to check php version before define `php_version` variable.

Maybe we should separate php5 and php7 to different tasks or roles to avoid many conditionals? I'm not sure about that, it will append duplicate code.